monitoring/backups_collector.go

```diff
@@ -0,0 +1,94 @@
+package monitoring
+
+import (
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+type BackupsCollector struct {
+	*baseCollector
+
+	sizeMetric  metricInfo
+	stateMetric metricInfo
+}
+
+func NewBackupsCollector(
+	logger logrus.FieldLogger,
+	nodeID string,
+	ds *datastore.DataStore) *BackupsCollector {
+
+	vc := &BackupsCollector{
+		baseCollector: newBaseCollector(subsystemBackups, logger, nodeID, ds),
+	}
+
+	vc.sizeMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemBackups, "actual_size_bytes"),
+			"Actual space used by each backup of the snapshot on the corresponding node",
+			[]string{nodeLabel, backupsLabel},
```

Comment:
Backup is a cluster-wide feature, so adding `nodeLabel` is not helpful.
But, it'd be good to add `volumeLabel` to indicate which volume's backup. And we are able to use volumeLabel to filter all the backups within this volume.
[1]

Reply:
Resolved on new commit ba6d2ee5add07fe03aae932d0c9423ed1a8e67c8